eng/common/testproxy/target_version.txt

```diff
@@ -1 +1 @@
-1.0.0-dev.20220805.3
+1.0.0-dev.20220810.2
```

Comment:
Requires a tag that includes https://github.com/Azure/azure-sdk-tools/commit/2279cc0fa00f1e5e4035236f4d6a6aee5dfe3902